### ISSUE TITLE
Building without handbook cloned locally fails

### ISSUE THREAD

**Issue by knolleary** [flowforge/website#26]
### Current Behavior

The build tasks assume the handbook repo has been checked out locally so its contents can be incorporated in the build.

That works fine for the GH Action as we make sure everything is there as part of the action.

When running locally, that cannot be assumed.

There is an attempt to handle this in the `dev:handbook` task:
```
"dev:handbook": "[ -e ../handbook ] && nodemon -w ../handbook -e md --exec 'npm run handbook' ",
```
But doesn't look like its working as intended.

The readme needs to explain this

### Expected Behavior

The website can be built locally without the handbook cloned locally.

**Comment by joepavitt** [comment]
forgot to include the issue number in the commit message, fix pushed to `issue--handbook-build` branch: 8cec36c33b17399cf9f78f7ca105b75dc605c182

**Comment by knolleary** [comment]
Fixed by #34
